aemdesign-aem-common/src/main/content/jcr_root/apps/aemdesign/clientlibs/core/vendorlib/bootstrap/js.txt

```diff
@@ -1,2 +1,2 @@
-js/bootstrap.min.js
-#js/bootstrap.bundle.js
+#js/bootstrap.min.js
+js/bootstrap.bundle.js
```

Comment:
Whats in bootstrap.bundle.js? Hopefully it doesn't conflict with anything in our dependencies @isoshaw